lib/ex_dtls.ex

```diff
@@ -92,7 +93,7 @@ defmodule ExDTLS do
   def init(opts) do
     {:ok, pid} = Unifex.CNode.start_link(:native)
     :ok = Unifex.CNode.call(pid, :init, [opts[:client_mode], opts[:dtls_srtp]])
-    state = %State{cnode: pid}
+    state = %State{cnode: pid, client_mode: opts[:client_mode]}
```

Comment:
Since these options are always required, let's use dot notation, like `opts.client_mode`

Reply:
It turned out that options are keyword list and dot syntax doesn't work for them

Reply:
use `Keyword.fetch!` then

Reply:
@mickel8